docs/development/xenial_support.rst

```diff
@@ -60,11 +80,6 @@ PAM logic
     ``pam_ecryptfs.so`` which prove problematic; commenting out ostensibly
     resolves. More research required.
 
-Config tests
-    The testinfra config test suite expects Trusty throughout. We'll need
-    to update that logic to refer to LSB codename instead, and assert
-    that the target platform is one of either Trusty or Xenial.
-
```

Comment:
Heads up: this paragraph was edited in #3833, so conflicts will arise. At first I also simply removed then, then decided to modify the language to encourage developers to support _both_ Trusty and Xenial in the config tests (as done, again, in #3833).

Reply:
OK, took the one from #3833